source/app/powertabeditor.cpp

```diff
@@ -229,7 +229,16 @@ void PowerTabEditor::openFile(QString filename)
     if (filename.isEmpty())
         return;
 
+	int validationResult = myDocumentManager->containsDocument(filename.toStdString());
```

Comment:
It looks like you might be using 8 spaces for indentation? The rest of the codebase uses 4 spaces.